src/utils/delegation.ts

```diff
@@ -0,0 +1,120 @@
+import { subgraphRequest } from '../utils';
+import delegationSubgraphs from '../delegationSubgraphs.json';
+
+export const SNAPSHOT_SUBGRAPH_URL = delegationSubgraphs;
+const PAGE_SIZE = 1000;
+
+type Delegation = {
+  delegator: string;
+  delegate: string;
+  space: string;
+  timestamp: number;
+};
+
+export default async function getDelegatesBySpace(
+  network: string,
+  space: string,
+  snapshot: string | number = 'latest',
+  options: any = {}
+) {
+  const subgraphUrl = options.subgraphUrl || SNAPSHOT_SUBGRAPH_URL[network];
+  if (!subgraphUrl) {
+    return Promise.reject(
+      `Delegation subgraph not available for network ${network}`
+    );
+  }
+
+  let result: Delegation[] = [];
+  const skip = 0;
+  const spaceIn = ['', space];
+  if (space.includes('.eth')) spaceIn.push(space.replace('.eth', ''));
+
+  while (true) {
+    const newResults = await fetchData({
+      url: subgraphUrl,
+      spaces: spaceIn,
+      first: PAGE_SIZE,
+      skip,
```

Comment:
Could we hard code these on `fetchData`?

Reply:
Sure, it was added for support of `skip` params in case of > 1000 duplicates, but can remove it